### ISSUE TITLE
[Question] Which properties to save/restore on rotation?

### ISSUE THREAD

**Issue by HubKing** [chrisbanes/PhotoView#611]
I could not find any information about that in the examples. I have seen basically the same question here, but there was no explanations but links to commits.

For example, I save/restore cameraPosition in order to keep the view state of a Google Map on rotation. I want to do a similar thing for PhotoView, but I am not sure what/how. I have tried to save/restore scale but it did not work. I am NOT asking for the general mechanism for saving/restoring values on rotation on the Android platform; I already know that. I am asking the properties of PhotoView, and the way to set those values to PhotoView.

In short, I want to keep the view state of PhotoView on screen rotation.

**Comment by HubKing** [comment]
After a little bit of testing, it seems that the scale must be set after image loading has been completed. I tried to restore the scale after a few seconds of delay and then it worked. Now the question is.... how do I know when the image loading has been completed?

		photoView.setImageResource(R.drawable.cat)
		photoView.attacher.update();
                //How to know `photoView` has done loading the image?
